.travis.yml

```diff
@@ -1,12 +1,35 @@
+---
 language: node_js
 node_js:
```

Comment:
[This](https://github.com/ember-cli/ember-cli/blob/master/.travis.yml) is what is listed in ember-cli's travis.yml, minus the now removed "node" line. I'd imagine you'd want to keep in sync with what ember-cli is doing